### DISCUSSION TITLE
[question] 请问是否可以：多个组件库（TinyVue, ElementPlus等）在tiny-engine中一起使用？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#879] [Q&A] [answered]
请问是否可以：多个组件库（TinyVue, ElementPlus等）在tiny-engine中一起使用呢？

![image](https://github.com/user-attachments/assets/e75ea217-2ad6-4403-9153-96777881358b)

**Comment by chilingling** [comment, marked as answer]
可以多个组件库一起使用。
但是正常的前端工程一般只会用到一个组件库就够了，没必要混合多个组件库，这里只是展示我们的能力。
